contrib/systemd/system/nordvpnd.service

```diff
@@ -6,6 +6,8 @@ Wants=network-online.target
 
 [Service]
 ExecStart=/usr/sbin/nordvpnd
+ExecStopPost=/bin/sh -c "iptables-save -c | grep -v '\\-\\-comment nordvpn' | iptables-restore -c"
```

Comment:
If the daemon crashes, wouldn't this result in leaks for ~5 seconds, until the OS restarts it?